libs/utils/utils.js

```diff
@@ -1184,3 +1184,11 @@ export function loadLana(options = {}) {
 }
 
 export const reloadPage = () => window.location.reload();
+
+export const setAttributes = (el, attrs) => {
+  Object.keys(attrs).forEach((key) => el.setAttribute(key, attrs[key]));
+};
+
+export const removeAttributes = (el, attrsKeys) => {
+  attrsKeys.forEach((key) => el.removeAttribute(key));
+};
```

Comment:
You probably guessed this was coming 🙈 We shouldn't add anything in utils unless it's used by a whole lot of places. Let's keep these where they're used for now and if we have a dire need for them in other places, we can reconsider the best location.

Reply:
Yup, the idea is to get some things out of utils, rather than into it at this point

Reply:
Totally agree! Moved them to the right place 👍